TOOLS.md

```diff
@@ -0,0 +1,49 @@
+# Tools
+
+| Name | Where binaries will be installed | Where will it be installed from | Supported platform | License |
+| ---- | -------------------------------- | ------------------------------- | ------------------ | ------- |
+| [**biome**](https://biomejs.dev) | `/usr/local/bin` | [GitHub Releases](https://github.com/biomejs/biome/releases) | Linux, macOS, Windows | [Apache-2.0](https://github.com/biomejs/biome/blob/main/LICENSE-APACHE) |
+| [**cargo-audit**](https://rustsec.org) | `$CARGO_HOME/bin` | [GitHub Releases](https://github.com/rustsec/rustsec/releases) | Linux, macOS, Windows | [Apache-2.0](https://github.com/rustsec/rustsec/blob/HEAD/cargo-audit/LICENSE-APACHE) OR [MIT](https://github.com/rustsec/rustsec/blob/HEAD/cargo-audit/LICENSE-MIT) |
+| [**cargo-binstall**](https://github.com/cargo-bins/cargo-binstall) | `$CARGO_HOME/bin` | [GitHub Releases](https://github.com/cargo-bins/cargo-binstall/releases) | Linux, macOS, Windows | [GPL-3.0](https://github.com/cargo-bins/cargo-binstall/blob/HEAD/crates/bin/LICENSE) |
+| [**cargo-careful**](https://github.com/RalfJung/cargo-careful) | `$CARGO_HOME/bin` | [GitHub Releases](https://github.com/RalfJung/cargo-careful/releases) | Linux, macOS, Windows | [MIT](https://github.com/RalfJung/cargo-careful/blob/master/LICENSE-MIT) OR [Apache-2.0](https://github.com/RalfJung/cargo-careful/blob/master/LICENSE-APACHE) |
+| [**cargo-cyclonedx**](https://cyclonedx.org/) | `$CARGO_HOME/bin` | [GitHub Releases](https://github.com/CycloneDX/cyclonedx-rust-cargo/releases) | Linux, macOS, Windows | [Apache-2.0](https://github.com/CycloneDX/cyclonedx-rust-cargo/blob/main/LICENSE) |
+| [**cargo-deny**](http://embark.rs) | `$CARGO_HOME/bin` | [GitHub Releases](https://github.com/EmbarkStudios/cargo-deny/releases) | Linux, macOS, Windows | [MIT](https://github.com/EmbarkStudios/cargo-deny/blob/main/LICENSE-MIT) OR [Apache-2.0](https://github.com/EmbarkStudios/cargo-deny/blob/main/LICENSE-APACHE) |
```

Comment:
Could you make it link to the repository instead of the website as before?

For cargo-deny, it would became annoying: auto-redirected and then have to scroll down the page to find the cargo-deny and click on it (finally jump to the cargo-deny repository). For cargo-cycloned, even the mention to the cargo-cycloned is not on the top page of cyclonedx.org.

Reply:
Is it ok if we do this specifically for cargo-deny and  cargo-cycloned ?

Or, what about I fetch the website and see if the crate name is mentioned, and if not fallback to the repo ?

btw, I am also happy to always use the repo - IMO most people find that most useful that a 'website'.  But the existing readme has website links for biome and others, which I why I tried to detect this

Reply:
I think it would be a better idea to make the default a repository link and allow the base manifest to specify a website link for biome and others.

(websites are sometimes created for the time being and have little or no information, or are abandoned.)

Another case where I felt the automatic detection of website links was not sufficient is cargo-audit: the current one is the link to the cargo-audit's readme, and the top page of repository and website have links to various tools, including links to the cargo-audit's readme, so it would be better to specify the current one than the repository link or the website link.

Reply:
(Btw, AFAIK biome, cargo-audit, shellcheck, and valgrind are the only ones currently not using repository links in main branch.)

Reply:
Remove the website auto-detection